controller/handler/trial.js

```diff
@@ -121,6 +121,11 @@ function trialView (request, reply) {
                     patient.totalMissed = 0;
                 }
 
+                patient.dateStarted = moment(patient.dateStarted)
+                    .format('MM-DD-YYYY');
```

Comment:
On patient's view, I had seen the format used as 'MM-DD-YYYY'.
I can change it if you want me to.